application/dash_application/views/preqc_wgs.py

```diff
@@ -482,14 +556,17 @@ def init_callbacks(dash_app):
             State(ids['second-sort'], 'value'),
             State(ids['colour-by'], 'value'),
             State(ids['shape-by'], 'value'),
+            State(ids["reads-per-start-point-slider"], 'value'),
+            State(ids["insert-mean-slider"], 'value'),
+            State(ids["passed-filter-reads-slider"], 'value'),
         ]
     )
     def update_pressed(click,
                        runs,
                        first_sort,
                        second_sort,
                        colour_by,
-                       shape_by):
+                       shape_by, total_reads_cutoff, insert_mean_cutoff, rpsp_cutoff):
```

Comment:
Can these be reformatted so they're one per line? It makes them easier to read :crying_cat_face:

Reply:
Sorry, I ran `autopep8`, but obviously not with enough aggression.